handlers.go

```diff
@@ -63,8 +63,11 @@ func (s *server) authenticate(w http.ResponseWriter, r *http.Request) {
 		if s.userIDOpts.tokenHeader != "" {
 			w.Header().Set(s.userIDOpts.tokenHeader, session.Values["idtoken"].(string))
 		}
+		logger.Debug("Existing user session "+userID)
 		returnStatus(w, http.StatusOK, "OK")
 		return
+	} else {
```

Comment:
I think this else is not needed.
After this point, you know it's a new session anyway

Reply:
The purpose of the else is to log the session struct.

Reply:
What I mean is that at that point, you know that this is a new session, because we return inside the if.
Thus, the else is not necessary, you could just log it.

Reply:
I understand now. Yes that's clearer.

Reply:
Updated now